src/imageio/format/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ cmake_minimum_required(VERSION 2.6)
 include_directories("${CMAKE_CURRENT_BINARY_DIR}/../../" "${CMAKE_CURRENT_SOURCE_DIR}")
 include_directories(SYSTEM "${PNG_PNG_INCLUDE_DIR}")
 
+set(CMAKE_C_VISIBILITY_PRESET hidden)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+
+add_definitions(-include modules_api.h)
```

Comment:
(it will be s/`modules_api.h`/`module_api.h`/)
not sure, i placed it in `src/modules_api.h`, and src/ is definitely in include path